idea-plugin/src/main/kotlin/org/jetbrains/compose/intentions/utils/isIntentionAvailable/IsIntentionAvailable.kt

```diff
@@ -0,0 +1,24 @@
+package org.jetbrains.compose.intentions.utils.isIntentionAvailable
+
+import com.intellij.psi.PsiElement
+import org.jetbrains.compose.intentions.utils.composableFinder.ComposableFunctionFinder
+import org.jetbrains.kotlin.idea.KotlinLanguage
+
+interface IsIntentionAvailable {
```

Comment:
Understood! That can be done, yes! 
I just personally dont like extension functions being available throughout the project. 

Since it's a single function, I'll do as you say